harvester_e2e_tests/apis/test_networks.py

```diff
@@ -99,3 +100,23 @@ def test_create_network_using_terraform(request, admin_session,
                                         harvester_api_endpoints,
                                         network_using_terraform):
     pass
+
+
+@pytest.mark.terraform
+@pytest.mark.public_network
+def test_cluster_network_using_terraform(request, admin_session,
+                                         harvester_api_endpoints):
```

Comment:
all you have to do is to add the `network` fixture in the args as we are already creating network using terraform.

```
def test_cluster_network_using_terraform(request, admin_session,
                                                                   harvester_api_endpoints, network):